website/static/css/style.css

```diff
@@ -337,6 +337,12 @@ a {
     margin-top: 4.8px;
 }
 
+/* Disable pointer events in page changing */
+.disabled {
```

Comment:
I'm concerned about this being defined in the site's main CSS file, but can't think of anywhere it would break.

@caneruguz, are you aware of the "disabled" class being used somewhere this would be a bad thing?